pom.xml

```diff
@@ -1,58 +1,141 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
 
     <parent>
         <groupId>org.openmrs.maven.parents</groupId>
         <artifactId>maven-parent-openmrs-module</artifactId>
         <version>1.1.1</version>
     </parent>
 
-	<groupId>org.openmrs.module</groupId>
-	<artifactId>ohricore</artifactId>
-	<version>1.0.0-SNAPSHOT</version>
-	<packaging>pom</packaging>
-	<name>OHRICore</name>
-	<description>OHRI Core openMRS module</description>
-
-	<url>https://wiki.openmrs.org/x/FQ4z</url>
-
-	<scm>
-		<connection>scm:git:git@github.com:openmrs/openmrs-module-ohricore.git</connection>
-		<developerConnection>scm:git:git@github.com:openmrs/openmrs-module-ohricore.git</developerConnection>
-		<url>https://github.com/openmrs/openmrs-module-ohricore/</url>
-	</scm>
-
-	<modules>
-		<module>api</module>
-		<module>omod</module>
-	</modules>
-
-	<repositories>
-		<repository>
-			<id>openmrs-repo</id>
-			<name>OpenMRS Nexus Repository</name>
-			<url>http://mavenrepo.openmrs.org/nexus/content/repositories/public</url>
-		</repository>
-		<repository>
-			<id>central</id>
-			<name>Maven Repository Switchboard</name>
-			<layout>default</layout>
-			<url>http://repo1.maven.org/maven2</url>
-		</repository>
-	</repositories>
-	<pluginRepositories>
-		<pluginRepository>
-			<id>openmrs-repo</id>
-			<name>OpenMRS Nexus Repository</name>
-			<url>http://mavenrepo.openmrs.org/nexus/content/repositories/public</url>
-			<snapshots>
-				<enabled>false</enabled>
-			</snapshots>
-		</pluginRepository>
-	</pluginRepositories>
+    <groupId>org.openmrs.module</groupId>
+    <artifactId>ohricore</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>OHRICore</name>
+    <description>OHRI Core openMRS module</description>
+
+    <url>https://wiki.openmrs.org/x/FQ4z</url>
+
+    <scm>
+        <connection>scm:git:git@github.com:openmrs/openmrs-module-ohricore.git</connection>
+        <developerConnection>scm:git:git@github.com:openmrs/openmrs-module-ohricore.git</developerConnection>
+        <url>https://github.com/openmrs/openmrs-module-ohricore/</url>
+    </scm>
+
+    <modules>
+        <module>api</module>
+        <module>omod</module>
+    </modules>
+
+    <dependencies>
+
+        <dependency>
```

Comment:
@smallgod The JUnit, PowerMock versions are included in the OpenMRS dependencies

Reply:
Removed!